client/db/src/offchain.rs

```diff
@@ -122,6 +118,16 @@ impl sp_core::offchain::OffchainStorage for LocalStorage {
 	}
 }
 
+/// Cocatenate the prefix and key to create an offchain key in the db.
```

Comment:
I am not totally sure we should make 'storage' + '/' + 'key' the default (to avoid breaking existing data).
I know we already discuss that, but I think current code does not insert '/' (except for indexing which was broken anyway), maybe I do not have look correctly (iiuc it is LocalStorage from client/db/src/offchain that is currently use with ocw).

Reply:
I can remove this, I don't care. However, I also don't see any harm in it.

Reply:
If the existing ocw data do not use it, we may break it?

Reply:
Removed